pkg/boot/fit/fit.go

```diff
@@ -132,7 +153,7 @@ func (i *Image) GetConfigName(verbose bool) (string, error) {
 
 // LoadConfig loads a configuration from a FIT image
 // Returns <kernel_name>, <ramdisk_name>, error
-func (i *Image) LoadConfig(verbose bool) (string, string, error) {
+func (i *Image) LoadConfig(skipinitfs bool, verbose bool) (string, string, error) {
```

Comment:
consider doing what our other packages do.
Create a package level variable
func Debug(string, ...interface{}) {}

Then users of the package can set this to, e.g., log.Printf

see how the bzimage package does it.

Reply:
Thanks for this. I realized it was better in the Image{}. Removed all the verboses as well. I went overboard trying to pass verbose down the chain.